flac.md

```diff
@@ -630,6 +630,28 @@ Decoding the coded residual thus involves selecting the right coding method, fin
 
 Following the last subframe is the frame footer. If the last subframe is not byte aligned (i.e. the bits required to store all subframes put together are not divisible by 8), zero bits are added until byte alignment is reached. Following this is a 16-bit CRC, initialized with 0, with polynomial x^16 + x^15 + x^2 + x^0. This CRC covers the whole frame excluding the 16-bit CRC, including the sync code.
 
+# Container mappings
+
+The FLAC format can be used without any container as the FLAC format already provides for a very thin transport layer. However, the functionalty of this transport is rather limited, and to be able to combine FLAC audio with video, it needs to be encapsulated by a more capable container. This presents a problem: the transport layer provided by the FLAC format mixes data that belongs to the encoded data (like blocksize and sample rate) with data that belongs to the transport (like checksum and timecode). The choice was made to encapsulate FLAC frames as they are, which means some data will be duplicated and potentially deviating between the FLAC frames and the encapsulating container.
+
+To differentiate encapsulated FLAC streams from non-encapsulated FLAC streams, the latter will be referred to as 'native FLAC'.
+
+## Ogg mapping
+
+The Ogg container format is defined in [@?RFC3533]. The first packet of a logical bitstream carrying FLAC data contains the bytes `0x7F 0x46 0x4C 0x41 0x43` as defined by [@?RFC5334], followed by two bytes coding the version number of the FLAC-in-Ogg mapping. These bytes are `0x01 0x00`, meaning version 1.0 of the mapping. Following the version number bytes is a 2-byte unsigned number coded big-endian containing the number of header packets (i.e. the number of metadata blocks) that follow the first packet.  This number MAY be 0, which means the number of packets that follow is unknown. Following the 2-byte number of header packets is the `fLaC` signature and a streaminfo metadata block. Taken together this means the first packet of an logical stream of FLAC-in-Ogg is always 79 bytes. This first packet MUST NOT share a Ogg page with any other packets.
+
+Following the first packet are one or more header packets i.e. metadata blocks. The first of these MUST be a vorbis comment metadata block. This is contrary to native FLAC streams, where the order of metadata blocks is not important except for the streaminfo block and where a vorbis comment metadata block is optional.
+
+Following the header packets are audio packets. Each FLAC frame is a packet. The first FLAC audio frame must start on a new Ogg page, i.e. the last metadata block must finish its page.
```

Comment:
"Each FLAC frame is a packet" -> "a OGG packet contains 1 (and only one?) FLAC frame"

Reply:
Maybe this?

```suggestion
Following the header packets are audio packets. Each audio packet contains a single FLAC frame. The first audio packet must start on a new Ogg page, i.e. the last metadata block must finish its page before any audio packets are encapsulated.
```

Reply:
Fine for me.